prophesee_ros_driver/src/prophesee_ros_publisher.cpp

```diff
@@ -71,18 +87,29 @@ PropheseeWrapperPublisher::PropheseeWrapperPublisher():
     auto &geometry = camera_.geometry();
     ROS_INFO("[CONF] Width:%i, Height:%i", geometry.width(), geometry.height());
     ROS_INFO("[CONF] Max event rate, in kEv/s: %u", config.max_drop_rate_limit_kEv_s);
+    ROS_INFO("[CONF] Streaming event array rate, in Hz: %4.2f [%lu microseconds]", this->event_streaming_rate_, this->event_delta_t_.toNSec()/1000);
+    ROS_INFO("[CONF] Activity Filter Temporal depth: %d [microseconds]", this->activity_filter_temporal_depth_);
     ROS_INFO("[CONF] Serial number: %s", config.serial_number.c_str());
 
     // Publish camera info message
     cam_info_msg_.width = geometry.width();
     cam_info_msg_.height = geometry.height();
     cam_info_msg_.header.frame_id = "PropheseeCamera_optical_frame";
+
+    // Set the activity filter instance
+    if (activity_filter_temporal_depth_ > 0) {
+    activity_filter_.reset(new Prophesee::ActivityNoiseFilterAlgorithm<>(camera_.geometry().width(),
```

Comment:
please, keep formatting

Reply:
Which formatting do you mean here? there is if (...)" "{ in line 100